test/Common/TestHelpers.cs

```diff
@@ -333,7 +333,8 @@ public static async Task WaitUntilTrue(Func<bool> predicate, string conditionDes
                 retryInterval = TimeSpan.FromMilliseconds(100);
             }
 
-            Stopwatch sw = new Stopwatch();
+            Stopwatch sw = Stopwatch.StartNew();
```

Comment:
Woops...

Reply:
Is this a comment to change something or just a woops :P